docs/developers_guide/framework/model.md

```diff
@@ -7,53 +7,51 @@
 Steps that run a standalone build of an E3SM component should descend from the
 {py:class}`polaris.ModelStep` class.
 
-By default (if the attribute `update_pio = True`), at runtime, the namelist 
-options associated with the [PIO library](https://ncar.github.io/ParallelIO/) 
-will automatically be set.  If the `make_graph = True`, the mesh will be 
+By default (if the attribute `update_pio = True`), at runtime, the namelist
+options associated with the [PIO library](https://ncar.github.io/ParallelIO/)
+will automatically be set.  If the `make_graph = True`, the mesh will be
 partitioned across MPI tasks at runtime.
 
-During construction or by setting attributes directly, you can can provide 
+During construction or by setting attributes directly, you can can provide
 non-default names for the graph, namelist and streams files.  At construction
-or with the {py:meth}`polaris.ModelStep.set_model_resources()` 
-method,  you can set number of tasks, threads, etc. determined from the 
-`ntasks`, `min_tasks`, `cpus_per_task`, `` min_cpus_per_task` `` and 
+or with the {py:meth}`polaris.ModelStep.set_model_resources()`
+method,  you can set number of tasks, threads, etc. determined from the
+`ntasks`, `min_tasks`, `cpus_per_task`, `` min_cpus_per_task` `` and
 `openmp_threads` attributes.  These resources need to be set at construction or
-in the  {ref}`dev-step-setup` method (i.e. before calling {ref}`dev-step-run`) 
-so that  the polaris framework  can ensure that the required resources are 
-available. If the graph partition file has been constructed prior to the ocean
-model step, the path to the graph file should be provided in the
-`graph_target` argument to {py:meth}`polaris.ocean.OceanModelStep.__init__()`.
```

Comment:
I moved this piece of the docs to the ocean framework, since it doesn't really belong in the general framework in my view.  (I also fixed the link to the OceanModelStep constructor.)